docs/src/Scenario2/Evaluation_Scenario_2.md

```diff
@@ -317,29 +326,6 @@ xmax, xmaxval = get_max_t(probv, sum(idart))
 
 This test passed with the original SBML model but failed with the model from the TA2 integration.
 
-### Setup the Parameters
-
-> Set the same initial values and parameter settings in 1.b.i. Let V(t=0) = 0, τ
-> (in SIDARTHE) = τ2 (in SIDDARTHE-V), and τ1 = (1/3)\*τ2 (reflecting the fact
-> that the mortality rate for critical conditions (state T), will always be
-> larger than for other infected states). Assume that the vaccination rate psi
-> is 0 to start with. The SIDARTHE-V model allows for three main types of
-> interventions: (1) Those that impact the transmission parameters (α, β, γ and
-> δ) – social distancing, masking, lockdown; (2) Those that impact the detection
-> parameters (ε, θ) – testing and contact tracing; (3) Those that impact the
-> vaccination rate psi – vaccination campaigns. Assume previously stated
-> constraints: θ >= 2* ε, and τ1 = (1/3)*τ2.
-
-```@example scenario2
-# TODO: tau1 and tau2?
-defs_v2 = copy(defaultsmapv)
```

Comment:
removing `defs_v2` would break everything after.

Reply:
Right. I think I meant to rename defs_v2 to defaultsmapv; is there a reason we need two versions of it?

Reply:
Not really. We were just setting the parameters after the model definition block before.